### DISCUSSION TITLE
Users with monthly subscriptions upgraded to lifetime users issue

### DISCUSSION THREAD

**Discussion by akump** [Glench/ExtPay#169] [General]
Hi,

I've had a few users say they have ran into issues when converting to a lifetime user. My workflow is
1. User has been a monthly subscriber for a few months
2. They decide to become lifetime. I go to "Add permanent free users" within extensionpay.com and input their email.
3. User cancels subscription.
4. When subscription cancel day occurs, their paid features apparently stop working.

I haven't been able to reproduce this using the dev setup. I was wondering if others have seen this.

Thanks

**Comment by Glench** [comment]
Hm, currently free users need to use the "log in" functionality to activate their paid features. Instead of trying to hack lifetime payments into the free user functionality, it would make much more sense for users to be able to upgrade themselves. This is coming, hopefully in the next few months.
